src/routes/safe/store/actions/createTransaction.js

```diff
@@ -94,6 +99,23 @@ const createTransaction = ({
   }
 
   try {
+    if (!isExecution && !isSmartContractWallet && semverSatisfies(safeVersion, '>=1.1.1')) {
+      const signature = await tryOffchainSigning({ ...txArgs, safeAddress })
+
+      if (signature) {
+        await saveTxToHistory({
+          ...txArgs,
+          signature,
+          type: TX_TYPE_CONFIRMATION,
```

Comment:
I think type is optional by now, and we could remove it (maybe create an issue for it, as it doesn'T affect this issue)

Reply:
@Uxio0 can you confirm that? API docs miss that info

Reply:
Confirmed @mikheevm, type will be removed and it's optional

Reply:
I see that I already removed it long time ago, that's why the API docs are not showing that. `type` was used before the big refactor last year.